doc/conf.py

```diff
@@ -58,6 +58,7 @@
     "sphinx.ext.linkcode",
     "sphinx.ext.autodoc.typehints",
     "sphinx_gallery.gen_gallery",
+    "sphinx_favicon",
```

Comment:
Do we need this extension? A favicon can be easily added in the jinja theme. One less extension is one less problem.

Reply:
True, we should remove it.